erizo_controller/erizoAgent/log4cxx.properties

```diff
@@ -1,5 +1,5 @@
 # Set root logger level to DEBUG and its only appender to A1.
-log4j.rootLogger=DEBUG, A1
+log4j.rootLogger=INFO, A1
```

Comment:
yes, it's an intentional change because I don't think we need to enable DEBUG level by default. But I can do it in a separate PR if we prefer.